src/scripts/plugin.js

```diff
@@ -9,15 +9,15 @@ const thebelabConfig = {
 const languageDictionary = {
   'Python 3': ['python'],
   Julia: ['julia'],
-  R: ['R'],
+  r: ['R'],
```

Comment:
This should be `R: ['r']`. The current code will map __line 121__ to `undefined`.

Reply:
Doing that gives ```Error!``` in the CKeditor